src/components/Stepper/index.js

```diff
@@ -22,80 +23,54 @@ const disabledStyles = ({ disabled, theme }) =>
   `
 
 const containerSizes = {
-  large: '48px',
-  medium: '40px',
-  small: '32px',
+  large: 48,
+  medium: 40,
+  small: 32,
 }
 
-const buttonSizes = {
-  large: '16px',
-  medium: '10px',
-  small: '4px',
+const iconSizes = {
+  large: 26,
+  medium: 24,
+  small: 22,
 }
 
 const StyledTouchable = styled(Touchable, {
   shouldForwardProp: prop => !['position', 'size'].includes(prop),
 })`
-  border: 1px solid ${({ theme }) => theme.colors.gray350};
-
   justify-content: center;
   align-items: center;
-  height: 100%;
-  padding-right: 16px;
-  padding-left: 16px;
+  height: calc(100% - 8px);
   position: relative;
+  border-radius: 4px;
+  width: ${({ size }) => containerSizes[size] - 10}px;
 
-  :hover,
-  :focus {
-    > svg {
-      fill: ${({ disabled, theme }) => !disabled && theme.colors.primary};
-    }
+  > svg {
```

Comment:
Possible to target the Icon instead pls ?

Reply:
Should be better @chambo-e